APIServer/api_endpoints.py

```diff
@@ -200,7 +200,15 @@ def get(self):
         This will return a dictionary of locations as keys
         and agent names as the value.
         """
-        return {"(0, 1)": "blue_grp0"}
+        exec_key = request.args.get('exec_key')
+        model = get_model(exec_key)
+        if model is None:
+            raise NotFound(f"Model Key: {exec_key}, not found.")
+            # not sure how to handle this error
+
+        model = model.to_json()
```

Comment:
Maybe use a different variable name to point to the json representation of the model?

Reply:
Good idea.